WowPacketParser/DBC/Structures/BattleForAzeroth/CreatureDisplayInfoEntry.cs

```diff
@@ -7,7 +7,7 @@ public sealed class CreatureDisplayInfoEntry
     {
         [Index(false)]
         public uint ID;
-        public ushort ModelID;
+        public ushort DisplayID;
```

Comment:
Looks like this was a text-based mass replace victim when you only wanted to change Creature member

Reply:
Nay. Afaik the true naming of that field is DisplayID and not ModelID and this is a remnant of doing it wrong back in the day. (UNIT_FIELD_DISPLAYID says hi)
Afaik he renamed it all intentionally.

Reply:
No, in CreatureDisplayInfo (dbc/db2), this field is a FK to CreatureModelData, and we have correct leaked names of almost all dbc (since wod) and db2 (since legion) fields

Reply:
Noted, when you finish review I will do all the issues in one go.

Reply:
That was an unintentional change.